### ISSUE TITLE
Finance Fee Waiver Report (TL)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#3302]
### As an ATD Finance Manager, I need a report that shows me ATD fees that have been waived, and totals for each waiver type. 

**Stakeholders:** Van P, Dan V
**DTS Manager:** Tracy

**Acceptance Criteria**
- ATD Fee Waivers - Finance report is available in ATD Custom Report Library
- Output is PDF
- User search parameters: Payment Start Date, Payment End Date
- Do not include refunds
- Only include fees that have been paid. 

- See report draft [here](https://github.com/cityofaustin/atd-data-tech/files/4892987/ATD.Fee.Waivers.-.Finance.xlsx)

![image](https://user-images.githubusercontent.com/49930868/89221417-dac04380-d598-11ea-88fd-f7c7bfc790d6.png)

![image](https://user-images.githubusercontent.com/49930868/89221488-f88da880-d598-11ea-9104-ba4014809fc1.png)

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan here is the list of GLs to include:


GL Account #
--
1000 5300 9770 4163
1000 5300 9770 4502
5090 5300 9300 4163
5125 2400 8790 8500
5125 2400 9100 4057
5125 2400 9100 4059
5125 2400 9100 4062
5125 2400 9100 4082
5125 2400 9100 4087
5125 2400 9100 4089
5125 2400 9100 4098
5125 2400 9100 4138
5125 2400 9100 4163
5125 2400 9100 4163
5125 2400 9100 4176
5125 2400 9100 4187
5125 2400 9100 4233
5125 2400 9100 4477
5125 2400 9100 4502
5125 2400 9100 4874
5125 2400 9770 4502
5126 2400 9100 4062
5126 2400 9100 4176
5127 2400 9100 4062
5127 2400 9100 4176
5128 2400 9100 4062
5128 2400 9100 4176
5129 2400 9100 4176
5130 2400 9100 4176
5610 2400 9200 4094
5610 2400 9200 4874

**Comment by TracyLinder** [comment]
**Status:** Work to begin in sprint 31.

**Comment by TracyLinder** [comment]
**Status:** This will be on hold until 8/27 due to staff resources

**Comment by Sreelatha-BlazeKannan** [comment]
@TracyLinder , @Nadin-Nader - Completed and Sent the Report for Review on 09/10/2020

**Comment by Nadin-Nader** [comment]
Reviewed report and it looks good @Sreelatha-BlazeKannan @TracyLinder

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader, @TracyLinder - DEV Deployment plan submitted on 09/22/2020

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder after Sree explained it a bit more it looks like everything is correct.

### Approved for TEST🎉

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder I have deployed this to TEST. Please log on and verify its all correct and let me know the outcome so we can have the stakeholders test and approve the move to PROD.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Required Tracy's & Nadin's Review feedback. TEST UAT is pending.

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan will do research to find out why test data is not populating.

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan can I please get a status update on this ticket? Thank you!

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Received reconfirmation on requirements from Tracy on 10/27/2020. Sree has to work on the report based on the requirements.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Sree will work on this when #1742 completed

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Work is in progress

**Comment by Sreelatha-BlazeKannan** [comment]
@TracyLinder @Nadin-Nader  - Waiting for feedback

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder Format and changes look good. Going to deploy to DEV and TEST.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder This has been deployed to DEV and TEST.

**Next Steps:**
UAT in TEST and get approval for PROD

**Comment by Sreelatha-BlazeKannan** [comment]
@TracyLinder @Nadin-Nader  - waiting for PROD approval

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan  @Nadin-Nader 

We were unable to UAT this in test this morning due to test issues; I'd still like to move this to PROD

### Approved for PROD 🎉

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder I have sent this over to the CTM team to Promote to PROD under TASK **SCTASK0105722**

## **Status:**
Waiting on CTM to promote to PROD

**Comment by TracyLinder** [comment]
This is done; closing issue;

**Comment by TracyLinder** [comment]
@Nadin-Nader can you please follow-up w/ CTM and find out why this isn't in prod?

**Comment by Nadin-Nader** [comment]
Sent an email to Jenice asking what happened to the report since it is missing in V7. She asked that i create a card on the V7 Trello Board. I asked her if she could give me access to the board. Waiting on response

**Comment by Nadin-Nader** [comment]
Created card on Trello under the Bugs Found column and added Jenice and Tracy to it. Jenice said she would take care of it. Waiting on Jenice to let me know when she fixes this

**Comment by Nadin-Nader** [comment]
Report is now good to go and on PROD.
